app/javascript/mastodon/components/admin/Counter.jsx

```diff
@@ -1,10 +1,15 @@
-import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import api from 'mastodon/api';
+import { PureComponent } from 'react';
+
+
 import { FormattedNumber } from 'react-intl';
-import { Sparklines, SparklinesCurve } from 'react-sparklines';
+
 import classNames from 'classnames';
-import { Skeleton } from 'mastodon/components/skeleton';
+
+import { Sparklines, SparklinesCurve } from 'react-sparklines';
+
+import api from 'mastodon/api';
+import {Skeleton} from 'mastodon/components/skeleton';
```

Comment:
The lack of spaces around `Skeleton` is surprising and inconsistent with the remaining of the code. What's causing this?

Reply:
My manual rebase before posting the PR, and the fact that this is not auto-fixed by Prettier.

Will go through them and fix this (same for newlines)